### PR TITLE
cpu/saml21: fixed and cleaned up startup.c

### PR DESCRIPTION
**Pull request by haukepetersen**
same as for the `samd21`

### CODE REVIEW

File: cpu/saml21/startup.c [hunk 1]
```diff
@@ -79,102 +80,80 @@ void reset_handler(void)
...
+void isr_tc1(void)              __attribute__ ((weak, alias("dummy_handler")));
+void isr_tc2(void)              __attribute__ ((weak, alias("dummy_handler")));
+void isr_tc3(void)              __attribute__ ((weak, alias("dummy_handler")));
+void isr_tc4(void)              __attribute__ ((weak, alias("dummy_handler")));
+void isr_adc(void)              __attribute__ ((weak, alias("dummy_handler")));
+void isr_ac(void)               __attribute__ ((weak, alias("dummy_handler")));
+void isr_dac(void)              __attribute__ ((weak, alias("dummy_handler")));
+void isr_ptc(void)              __attribute__ ((weak, alias("dummy_handler")));
+void isr_aes(void)              __attribute__ ((weak, alias("dummy_handler")));
+void isr_trng(void)             __attribute__ ((weak, alias("dummy_handler")));
```
Comment (kaspar030) [inline cpu/saml21/startup.c]: this is how you get your high line counts. :)

Comment (haukepetersen) [inline cpu/saml21/startup.c]: **ck the line counts, but when I see something that is keeping to the 80 char line length and it is soo easy to fix I can just not ignore it...

**Comment by haukepetersen** [comment]
rebased.

**Comment by kaspar030** [comment]
compiles, runs, ACK&go.

**Pull request merged**